frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -872,6 +872,14 @@
             scope.sortColumn = column;
         };
 
+        vm.format_execution_time = function(execution_time) {
+            var execution_time_hr = Math.floor(execution_time/60/60);
+            var execution_time_min = Math.floor(execution_time/60) - (execution_time_hr * 60);
+            var execution_time_sec = Math.floor(execution_time % 60);
```

Comment:
@Alabhya268 can we use date util to do the conversion. See [this](https://stackoverflow.com/questions/1322732/convert-seconds-to-hh-mm-ss-with-javascript) for reference

Reply:
@Ram81, I tried to do it in a way . It not working as expected.
Its adding the executions time in current time. I tried to modify it , I am able to get hour, min and sec but not days with this method.